editor/editor_help.cpp

```diff
@@ -1971,9 +1969,11 @@ EditorHelpBit::EditorHelpBit() {
 	add_child(rich_text);
 	rich_text->connect("meta_clicked", callable_mp(this, &EditorHelpBit::_meta_clicked));
 	rich_text->set_override_selected_font_color(false);
-	set_custom_minimum_size(Size2(0, 70 * EDSCALE));
```

Comment:
I changed my mind and added it back, since it would cause this on wrongly initialized entries:
https://github.com/godotengine/godot/pull/57547#issuecomment-1028078230